srcpkgs/vala-panel-appmenu/template

```diff
@@ -0,0 +1,91 @@
+# Template file for 'vala-panel-appmenu'
+pkgname=vala-panel-appmenu
+version=0.7.1
+revision=1
+build_style=cmake
+configure_args="-DENABLE_XFCE=ON -DENABLE_VALAPANEL=ON
+ -DENABLE_MATE=ON -DENABLE_JAYATANA=ON -DENABLE_APPMENU_GTK_MODULE=ON
+ -DJAVA_INCLUDE_PATH=/usr/lib/jvm/openjdk/include
+ -DJAVA_JVM_LIBRARY=/usr/lib/jvm/openjdk/lib
+ -DJAVA_AWT_LIBRARY=/usr/lib/jvm/openjdk/lib/amd64"
+hostmakedepends="pkg-config"
+makedepends="vala-devel bamf-devel gtk+-devel gtk+3-devel glib-devel
+ libxkbcommon-devel openjdk libpeas-devel libdbusmenu-glib-devel cmake-vala
+ xfce4-panel-devel xfconf-devel libmate-panel-devel libXt-devel"
+depends="glib bamf "
+short_desc="Global Menu for Vala Panel"
+maintainer="Juan RP <xtraeme@voidlinux.org>"
+license="LGPL-3.0-or-later"
+homepage="https://gitlab.com/vala-panel-project/vala-panel-appmenu"
+distfiles="${homepage}/uploads/c0f85b42a2a228ad8c3413357e02fb28/vala-panel-appmenu-${version}.tar.xz"
+checksum=54a1b4f6ceca63eb3b79facc4327424bdd82f0fe3b7c6139326dbe0ec0824dcc
+subpackages="appmenu-gtk-module-devel appmenu-gtk-module appmenu-gtk3-module
+ appmenu-jayatana appmenu-registrar xfce4-panel-appmenu mate-panel-appmenu"
+
+appmenu-gtk-module-devel_package() {
+	short_desc="GTK module for exposing menus - development files"
+	depends+="appmenu-gtk3-module"
+	pkg_install() {
+		vmove usr/include
+	}
+}
+
+appmenu-gtk-module_package() {
+	short_desc="GTK module for exposing menus"
+	depends+="gtk+"
+	pkg_install() {
+		vmove usr/lib/pkgconfig/appmenu-gtk2-parser.pc
+		vmove usr/lib/gtk-2.0
+		vmove usr/lib/libappmenu-gtk2*
+	}
+}
+
+appmenu-gtk3-module_package() {
+	short_desc="GTK3 module for exposing menus"
+	depends+="gtk+3"
+	pkg_install() {
+		vmove usr/lib/pkgconfig/appmenu-gtk3-parser.pc
+		vmove usr/lib/gtk-3.0
+		vmove usr/lib/libappmenu-gtk3*
+	}
+}
+
+appmenu-jayatana_package() {
+	short_desc="Global Menu for Java Swing applications"
+	depends+="openjdk libxkbcommon libdbusmenu-glib"
+	pkg_install() {
+		vmove usr/share/java
+		vmove usr/lib/jayatana
+	}
+}
+
+appmenu-registrar_package() {
+	short_desc="Appmenu registrar that allows to access a window's menu tree"
+	depends="glib"
+	pkg_install() {
+		vmove usr/libexec/vala-panel
+		vmove usr/share/dbus-1
+	}
+}
+
+xfce4-panel-appmenu_package() {
+	short_desc="XFCE Panel applet for appmenu module"
+	depends+="xfce4-panel"
+	pkg_install() {
+		vmove usr/lib/xfce4
+		vmove usr/share/xfce4
+	}
+}
+
+mate-panel-appmenu_package() {
+	short_desc="Mate Panel applet for appmenu module"
+	depends+="mate-panel"
+	pkg_install() {
+		vmove usr/lib/mate-panel
+		vmove usr/share/mate-panel
+	}
+}
+
+vala-panel-appmenu_package() {
+	depends+=" vala-panel"
```

Comment:
what ?

Reply:
vala-panel-appmenu needs libvalapanel from vala-panel

Reply:
you are defining vala-panel-appmenu as a subpackage while it is also the main package.

Reply:
ups i will correct that